src/main/groovy/edu/oregonstate/mist/personsapi/PersonsResource.groovy

```diff
@@ -794,4 +795,29 @@ class PersonsResource extends Resource {
             internalServerError("Unable to execute SQL query").build()
         }
     }
+
+    @Timed
+    @GET
+    @Path('{osuID: [0-9]+}/phones')
+    Response getPhones(@PathParam('osuID') String osuID,
+                       @QueryParam('addressType') String addressType,
+                       @QueryParam('phoneType') String phoneType) {
+        if (bannerPersonsReadDAO.personExist(osuID)) {
+            List<PhoneObject> phones = bannerPersonsReadDAO.getPhones(osuID, addressType, phoneType)
```

Comment:
We also wanna validate if `addressType` and `phoneType` are valid.

Reply:
This validation is not done in `getAddresses`. Would you like me to add that there as well?

Reply:
Oh I didn't notice, yeah, feel free to add it if you don't mind, thanks 👍